src/main/java/com/faforever/client/chat/ChatUserItemController.java

```diff
@@ -312,25 +316,37 @@ private void updateGameStatus() {
     }
     Optional<Player> playerOptional = chatUser.getPlayer();
     if (!playerOptional.isPresent()) {
-      statusLabel.setText("");
+      playerStatusIndicator.setVisible(false);
+      playerMapIndicator.setVisible(false);
       return;
     }
 
     Player player = playerOptional.get();
     switch (player.getStatus()) {
       case IDLE:
-        statusLabel.setText("");
+        playerStatusIndicator.setVisible(false);
```

Comment:
If we do it at initialize will it update?

Reply:
what do you mean?